src/components/HorizontalTab/ChannelListTabItem.tsx

```diff
@@ -74,4 +76,4 @@ const ChannelListTabItem = (props: ChannelListTabItemProps) => {
   );
 };
 
-export default ChannelListTabItem;
+export default React.memo(ChannelListTabItem);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Wrapping the `ChannelListTabItem` component in `React.memo` is a good practice for performance optimization, as it will prevent unnecessary re-renders if the props haven't changed. However, keep in mind that `React.memo` does a shallow comparison of props, so it might not work as expected if the props are complex objects.
```diff
- export default ChannelListTabItem;
+ export default React.memo(ChannelListTabItem);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->